main/admin/init_cas_user_extra_field_from_ldap.php

```diff
@@ -0,0 +1,323 @@
+<?php
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [End of line character is invalid; expected "'\n'" but found "\n"](https://app.codacy.com/manual/chamilo/chamilo-lms/pullRequest?prid=4906847)

Reply:
false positive (well, it is true that end of line is not "'\n'" but that it a bug in the test)